benchmarks/backgrounds/Snakefile

```diff
@@ -0,0 +1,107 @@
+import os
+import shutil
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+
+S3 = S3RemoteProvider(
+    endpoint_url="https://eics3.sdcc.bnl.gov:9000",
+    access_key_id=os.environ["S3_ACCESS_KEY"],
+    secret_access_key=os.environ["S3_SECRET_KEY"],
+)
```

Comment:
For my information, is there an xrootd provider? Can we use that for an example to get some variety here?

Reply:
Yes. The xrootd will be added soon.